common/config/qryn.go

```diff
@@ -101,5 +149,22 @@ func parseURL(rawURL, apiKey, apiSecret string) (string, error) {
 		return parseURL(fmt.Sprintf("https://%s", rawURL), apiKey, apiSecret)
 	}
 
```

Comment:
Consider trimming rawURL white spaces, as it will make `url.Parse(rawURL)` return with error.

```
	noWhiteSpaces := strings.TrimSpace(rawUrl)
	parsedUrl, err := url.Parse(noWhiteSpaces)
```